matflow_damask/main.py

```diff
@@ -341,6 +356,60 @@ def modify_volume_element_add_buffer_zones(volume_element, buffer_sizes,
     return out
 
 
+@func_mapper(task='modify_volume_element', method='spread_orientations')
+def modify_volume_element_spread_orientations(volume_element, phase, max_spread_angle):
+    """
+    Include additional orientations within a limit around the orientation of a given phase.
+
+    append new idx and phase label to idx and phase label arrays, calculate oris within max range from given ori, append them to ori array. 
+    """
+    n_samples = 100
```

Comment:
For now `n_samples` (number of orientations to sample around defined orientation) is arbitrarily 100. Should this be a user-defined variable?

Reply:
Understood. This was fine but would have run into a problem if the user tried to sample more orientations than there are voxels assigned to the phase in the VE. I will change n_samples to equal the number of material points assigned to the user-defined phase in the VE.